src/sentry/api/bases/organization_events.py

```diff
@@ -108,72 +102,3 @@ def get_snuba_query_args_legacy(self, request, organization):
             )
 
         return snuba_args
-
-    def next_event_id(self, snuba_args, event):
```

Comment:
What's the reason for moving these methods into OrganizationEventDetailsEndpoint ? Were they not used anywhere else ?

Reply:
Thanks for doing this cleanup :+1: